internal/config/client.go

```diff
@@ -20,14 +20,16 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/logging"
 	"github.com/mongodb-forks/digest"
+	adminpreview "github.com/mongodb/atlas-sdk-go/admin"
 	"github.com/spf13/cast"
 
 	"github.com/mongodb/terraform-provider-mongodbatlas/version"
 )
 
 const (
-	toolName              = "terraform-provider-mongodbatlas"
-	terraformPlatformName = "Terraform"
+	toolName                             = "terraform-provider-mongodbatlas"
+	terraformPlatformName                = "Terraform"
+	previewV2AdvancedClusterEnabledUAKey = "IsAdvancedClusterPreview"
```

Comment:
```suggestion
	previewV2AdvancedClusterEnabledUAKey = "AdvancedClusterPreview"
```

very nit, just shorter

Reply:
done